databases/week1/week1.txt

```diff
@@ -0,0 +1,32 @@
+1)Find out how many tasks are in the task table
+--SELECT count(id) AS count_task  FROM task;
+
+2)Find out how many tasks in the task table do not have a valid due date
+--SELECT count(id) AS  "number of tasks with invalid due date" FROM task WHERE due_date IS null;
+
+3)Find all the tasks that are marked as done
+--SELECT * FROM task WHERE status_id = 3;
+
+4)Find all the tasks that are not marked as done
+--SELECT * FROM task WHERE status_id != 3;
+
+5)Get all the tasks, sorted with the most recently created first
+--SELECT * from task ORDER BY CREATED DESC ;
+
+6)Get the single most recently created task
+--SELECT * from task ORDER BY CREATED DESC LIMIT  1 ;
+
+7)Get the title and due date of all tasks where the title or description contains database
+--SELECT title,due_date FROM task WHERE title LIKE '%DATABASE%' or DESCRIPTION LIKE '%DATABASE%';
+
+
+8)Get the title and status (as text) of all tasks
+--SELECT title, status.name FROM task INNER JOIN status ON status.id = status_id;
+
+9)Get the name of each status, along with a count of how many tasks have that status
+--SELECT STATUS.NAME,count(task_id) FROM task  INNER JOIN STATUS ON TASK.STATUS_ID = STATUS.ID ;
```

Comment:
This one gives my and error `Error in query (1146): Table 'hyf_lesson1.STATUS' doesn't exist`. This is because you have written some of the table names in uppercase. Also the query is not correct when i fix the mentioned error. You basically already have the correct query in exercise 10, just without the ORDER BY.

Reply:
Thank you for giving the feedback. I have implemented the changes and pushed the homework again. I have  renamed the column names  to lowercase and put comments for the questions
https://github.com/ajgcph2019/hyf-homework/pull/15/commits/0417064ebecfae1de940eeb9c9bf961d92b0a10a